pylint_odoo/test_repo/broken_module/model_view2.xml

```diff
@@ -104,6 +104,23 @@
             <field name="name">view.model.form80</field>
             <field name="model">test.model</field>
         </record>
-
+        <!-- Replace with low priority wo field -->
+        <record id="view_model_form90" model="ir.ui.view">
+            <field name="name">view.model.form90</field>
+            <field name="model">test.model</field>
+            <field name="priority">10</field>
+            <field name="arch" type="xml">
+                 <label for="name" position="replace"/>
+            </field>
+        </record>
+        <!-- Record with position different to replace -->
+        <record id="view_model_form100" model="ir.ui.view">
+            <field name="name">view.model.form100</field>
+            <field name="model">test.model</field>
+            <field name="priority">10</field>
+            <field name="arch" type="xml">
+                 <label for="name" position="after"/>
```

Comment:
- Include all positions: before, after, attributes and move (move must go inside another)
- I haven't checked if there are views already containing those. Could you check, please? If so, maybe you can reuse it, in case they include only valid cases, i.e. they don't trigger any lint. Sorry for not commenting this before

Reply:
I checked it and it does not contain tests with all positions. I added test cases with positions that you commented me.